### ISSUE TITLE
ui: Style updates to database page

### ISSUE THREAD

**Issue by Annebirzin** [cockroachdb/cockroach#45725]
- [ ] Since we are reverting back to current database page design, we can make minor style updates to align with the rest of the admin ui styles: [Figma designs ](https://www.figma.com/file/Tzrm57PT9S9Y8YtkTKejIT/obsrv_admin-ui-databases?node-id=1357%3A4885)
![Databases_00](https://user-images.githubusercontent.com/51920721/75921363-fd560100-5e2e-11ea-887a-1b2a52d33ddf.png)

**Comment by piyush-singh** [comment]
Vlad noted that this is done but blocked by another PR
